docs/release-notes/next-lima/PT-165214367_offchain_pinned_env.md

```diff
@@ -0,0 +1,4 @@
+* Enhances the FSM with the optional pinned environment for execution
```

Comment:
Is the `pinned environment` described in more detail somewhere else which could be linked here?